### DISCUSSION TITLE
CTM Sample App release to App Store and Google Play

### DISCUSSION THREAD

**Discussion by GUOLDEV** [CommunityToolkit/Maui#187] [General]
@jfversluis @pictos @bijington CTM Sample App release to App Store and Google Play, I am happy to do that using my private account, I just need the green light from you guys.

**Comment by TheCodeTraveler** [comment]
Hi @GUOLDEV! 

Interesting idea, although I'm not too keen on it for a couple reasons:
- The Sample App is not production-quality
  - It needs a significant UI overhaul to be a polished App Store app
- We would need to publish it under a Microsoft account or a CommunityToolkit 
- Value Added vs Maintenance Cost?
  - I'm not sure the work required to maintain an App Store App adds enough value to outweigh the maintenance cost of publishing it and keeping it up-to-date, but I'm happy to be wrong on this one!

**Reply by GUOLDEV** [reply]
Hi @brminnick thanks for the prompt reply! 
👍 I totally understand the reasons that you pointed. 
☝️ Can I publish it under my account if I clearly say in title for example "not official"? 
👀 So it's going to be just one more option for people to play with it and maybe start using it. 
🧐 Thoughts?

**Reply by jfversluis** [reply]
I agree with Brandon, we have talked about this on many occasions also for the Xamarin Community Toolkit. For instance here: https://github.com/xamarin/XamarinCommunityToolkit/discussions/241

Additionally I think we should either do it through official channels or not do it. I appreciate the offer, but I do have my own personal accounts as well so then I guess it would make more sense for me to do it. I can't really forbid you to do it, but I would prefer if you didn't as it will probably only cause confusion.

Like Brandon said; if we were even to consider this I think we would definitely make a better looking app so that it also helps our goal. If we publish as-is, I think people will look at it and think: this is not great. While, if you have some more design skills, you can make something great out of it!

While some people have been asking for it now, I personally don't really understand. I appreciate the gesture, but if you're looking into this I assume that you are a developer. Why is it so much harder to pull down the code and run the sample app yourself? That will also give you insights in how it all works and play with it then and there.

**Reply by GUOLDEV** [reply]
Hi @jfversluis I really appreciate your response and explanation.

Yes, I'm a dev and I have XCT sample installed in my test devices, I use it a lot to discuss with my team members what we can implement next in our product.

I do use other sample apps as reference as well, example Telerik UI sample app, Grial UIKit etc.

These sample apps are not that great in terms of design, but it helps a lot to decide what to do when you need to implement something different.

As dev, I think a sample app is a very important tool to demonstrate what you can do with that package, getting more people interested in working with it and that means more people interested in helping improving it. 

It's a win win in my opinion.

About releasing the app under my private account, it's just my way trying to help as I know you guys are already busy enough with more important things and decisions.

Thanks again for the hard work guys 👍

**Reply by jfversluis** [reply]
To be very clear: I appreciate the offer! :) Thank you for that!

**Comment by VladislavAntonyuk** [comment]
maybe we can use AppCenter with Public Distribution Group or provide the link to the Artifacts on Azure. it is not exactly what you requested, but it will allow testing apps without the necessity to build from the specific branch.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
